torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -662,7 +356,7 @@ void PrepareCopyForONNX(Node* node) {
     expanded_value->copyMetadata(node->input(1));
 
     auto index_put = graph->insert(
-        aten::index_put,
+        aten::index_put_,
```

Comment:
Shall we delete the unused copy_ here at the end of this function?

Reply:
i feel it maybe better leaving this to dce. otherwise it will depend on outer loop to correctly manage the iteration pointer. And further there may be rare cases where new nodes are added after this node, the iteration pointer will end up pointing to nodes beyond that.

Reply:
Oh I see. Thanks.
Is there a pointer used in graph->insert API?